src/main/java/step3/ManualLotto.java

```diff
@@ -0,0 +1,21 @@
+package step3;
+
+public class ManualLotto {
+    private static final String MANUAL_LOTTO_NUM_ERROR = "수동으로 구매할 수 있는 로또의 수는 구입하는 로또의 수 보다 적어야합니다.";
+    private final int numOfManualPurchaseLotto;
```

Comment:
```suggestion
    private static final String MANUAL_LOTTO_NUM_ERROR = "수동으로 구매할 수 있는 로또의 수는 구입하는 로또의 수 보다 적어야합니다.";
    
    private final int numOfManualPurchaseLotto;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다 :)

Reply:
넵!! 수정하였습니다!